src/sentry/sentry_metrics/consumers/indexer/processing.py

```diff
@@ -122,9 +122,7 @@ def _process_messages_impl(
         sdk.set_measurement("org_strings.len", len(org_strings))
 
         with metrics.timer("metrics_consumer.bulk_record"), sentry_sdk.start_span(op="bulk_record"):
-            record_result = self._indexer.bulk_record(
-                use_case_id=self._config.use_case_id, org_strings=org_strings
-            )
+            record_result = self._indexer.bulk_record({self._config.use_case_id.value: org_strings})
```

Comment:
Let's add a note on this PR to make it clear that this is temporary and is waiting for https://github.com/getsentry/sentry/pull/47113